src/invoices/InvoicesList/Invoice/InvoiceLine.js

```diff
@@ -0,0 +1,67 @@
+import React, { Component } from 'react';
```

Comment:
I'd moved these components under `InvoiceLine` or smth like that

Reply:
in my opinion it's coupled to Invoice tightly, at least with `parentMutator` and `parentResources`. Does it make sense to leverage this into separate module?